reconcile/dashdotdb_base.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import os
+
+from urllib.parse import urljoin
+
+import requests
+
+from reconcile import queries
+from reconcile.utils.secret_reader import SecretReader
+
+LOG = logging.getLogger(__name__)
+
+DASHDOTDB_SECRET = os.environ.get('DASHDOTDB_SECRET',
+                                  'app-sre/dashdot/auth-proxy-production')
+
+
+class DashdotdbBase:
+    def __init__(self, dry_run, thread_pool_size, marker):
+        self.dry_run = dry_run
+        self.thread_pool_size = thread_pool_size
+        self.settings = queries.get_app_interface_settings()
+        self.secret_reader = SecretReader(settings=self.settings)
+        self.secret_content = self.secret_reader.read_all(
+            {'path': DASHDOTDB_SECRET})
+        self.dashdotdb_url = self.secret_content['url']
+        self.dashdotdb_user = self.secret_content['username']
+        self.dashdotdb_pass = self.secret_content['password']
+        self.logmarker = marker
```

Comment:
I think we could init and verify the scope and set to self.scope here

Reply:
Yes, good idea.  Done